config/default.yml

```diff
@@ -1956,6 +1956,13 @@ Lint/NoReturnInBeginEndBlocks:
   Enabled: pending
   VersionAdded: '1.2'
 
+Lint/NonAtomicFileOperation:
+  Description: Checks for non-atomic file operations.
+  Enabled: pending
+  VersionAdded: '<<next>>'
+  Safe: false
```

Comment:
I think this `Safe: false` can be removed because it seems likely that false positives will not occur.

```suggestion
```